scripts/aggregate-stability.py

```
@@ -0,0 +1,437 @@
+        lines.append("")
+        lines.append(f"Runs: **{num_runs}** | "
+                      f"Suite passed: **{mode_result['run_passes']}/{num_runs}** | "
+                      f"Avg run duration: **{format_duration(mode_result['avg_run_duration'])}**")
+        lines.append("")
+
+        lines.append("### Summary")
+        lines.append("")
+        lines.append("| Metric | Value |")
+        lines.append("|--------|-------|")
+        lines.append(f"| Total tests | {total} |")
+        pct = lambda n: f"{n/total*100:.0f}%" if total else "0%"
```

Ruff (0.15.17)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `pct` as a `def`

(E731)